docs/docs/training-data-format.mdx

```diff
@@ -100,7 +98,7 @@ stories:
     intent: faq/language
   - action: utter_faq
 ```
-They use the same format as the story training data.
+Test stories use the same format as the story training data.
```

Comment:
L89: link to test stories section, and be expicit that the name of the separate file needs to start with `test_

Reply:
also "If you want to" -> "To"